official/boosted_trees/train_higgs_test.py

```diff
@@ -73,8 +73,13 @@ def test_make_inputs_from_np_arrays(self):
     train_data, _ = train_higgs.read_higgs_data(
         self.data_dir,
         train_start=0, train_count=15, eval_start=15, eval_count=5)
-    input_fn, feature_columns = train_higgs.make_inputs_from_np_arrays(
-        features_np=train_data[:, 1:], label_np=train_data[:, 0:1])
+    (input_fn, feature_names,
+     feature_columns) = train_higgs.make_inputs_from_np_arrays(
+         features_np=train_data[:, 1:], label_np=train_data[:, 0:1])
+
+    # Check feature_names.
+    self.assertAllEqual(feature_names,
+                        ['feature_%02d' % (i+1) for i in range(28)])
```

Comment:
nit: use double quotes to keep consistent. ["feature_%02d" % (i + 1) for i in range(28)]

Reply:
Done.